### PR TITLE
support lazy loading of images

### LINKED ISSUE #15: lazy image loading support
I am a frequent visitor to https://government.github.com/community/, which uses this plugin to display the avatars. The page takes 6.7 seconds to load on my relatively fast connection/machine, because there are 1065 images being requested.

In short, it would be great for jekyll-avatar to support lazy image loading. I've looked at [a](https://www.andreaverlicchi.eu/lazyload/) [few](https://appelsiini.net/projects/lazyload/) [plugins](https://luis-almeida.github.io/unveil/), and they all use the image URL in a `data-src` attribute (rather than `src`). Open to a pull request to support that?

cc @jbjonesjr

### PR DESCRIPTION
**Pull request by afeld**
Closes #15. Builds on #17 - [compare](https://github.com/afeld/jekyll-avatar/compare/test-improvements...lazy-load).

This pull request adds an option for supporting lazy loading of images via a JavaScript plugin. Tried these changes out on https://government.github.com/community/ locally with the [vanilla-lazyload](https://www.andreaverlicchi.eu/lazyload/) plugin. Before, on initial page load:

screen shot 2018-03-31 at 10 59 03 am

After:

screen shot 2018-03-31 at 10 58 19 am

Note that the images won't show up if JavaScript isn't available. It's possible to [get around this with a `<noscript>` fallback](https://www.robinosborne.co.uk/2016/05/16/lazy-loading-images-dont-rely-on-javascript/#a-no-JavaScript), but that would require a more dramatic change to the rendered HTML, which would be a breaking change for users who are expecting the existing structure. Your call!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,11 +73,24 @@ Or, if the variable is someplace a bit more complex, like a loop:
 {% endfor %}
 ```
 
+### Lazy loading images
+
+For pages showing a large number of avatars, you may want to load the images lazily.
+
+```liquid
+{% avatar hubot lazy=true %}
```
Comment (benbalter) [inline README.md]: Nice. Is there an advantage to making the flag per-tag versus one global flag? Would users want some avatars to be lazy loaded and others to be loaded in-line?

Comment (afeld) [reply README.md]: Umm, I don't feel super strongly either way. This seems more flexible, so I would err on this side.

Comment (benbalter) [reply README.md]: 👍 Easy enough to make it global later if people want it.

**Comment by benbalter** [comment]
Thanks @afeld! Corrected the Rubocop offenses locally (single versus double quotes) and merged. 🎉 💥 🌮

**Pull request merged**